### PR TITLE
Exclusively build for pwdata and filter out modules 

### PR DESCRIPTION
**Pull request by jacksongoode**
This way we can pass in `["pwdata"]` as a exclusive package to filter by and then skip all the Pipeline classes as `ignored_mods`.

### CODE REVIEW

File: pycg/processing/preprocessor.py [hunk 1]
```diff
@@ -205,6 +215,14 @@ def add_external_def(name, target):
             tgt_name = import_item.asname if import_item.asname else import_item.name
             imported_name = self.import_manager.handle_import(src_name, level)
 
+            # Limit to exclusive module
+            if src_name.split(".")[0] not in self.exclusives:
+                continue
+
+            # Skip ignored modules
```
Comment (adkinsjd) [inline pycg/processing/preprocessor.py]: Why do we need to skip ignored modules down here and remove them from times above?

File: pycg/processing/preprocessor.py [hunk 2]
```diff
@@ -205,6 +215,14 @@ def add_external_def(name, target):
             tgt_name = import_item.asname if import_item.asname else import_item.name
             imported_name = self.import_manager.handle_import(src_name, level)
 
+            # Limit to exclusive module
+            if src_name.split(".")[0] not in self.exclusives:
```
Comment (adkinsjd) [inline pycg/processing/preprocessor.py]: What happens here if self.exclusives is None?

**Review by adkinsjd — COMMENTED** [review]
A few small comments. I just want to make sure I understand what's happening here.

File: pycg/processing/preprocessor.py (hunk 2 above)
Comment (jacksongoode) [reply pycg/processing/preprocessor.py]: Good catch, I'll add their default to be an empty array.

File: pycg/processing/preprocessor.py (hunk 1 above)
Comment (jacksongoode) [reply pycg/processing/preprocessor.py]: Below is what happens when PyCG looks at an import declaration, the other `... in self.ignored_mods:` case above is when we visit a module. I did this as a safety precaution more so I think PyCG could end up traversing through a ignored modules in both cases - though I could be wrong about this and we can test.

**Pull request merged**